### ISSUE TITLE
Failing to connect even though password has been provided

### ISSUE THREAD

**Issue by ShayNehmad** [tableau-mkt/github-data-connector#18]
Followed the instructions and got:
```
An error occurred while communicating with Web Data Connector.

Bad Connection: Tableau could not connect to the data source.
Error Code: 37CE01A3
Missing password.
```

Obviously I've provided a password. 

What's happening and how can I solve it?

**Comment by Emiya0306** [comment]
+1, I think maybe it is tableau's problem. Because the WDC I can run before, and today, the WDC don't work...

**Comment by Emiya0306** [comment]
+1, I think maybe it is tableau's problem. Because the WDC I can run before, and today, the WDC don't work...

**Comment by Etroid** [comment]
@ShayNehmad After authenticating are you presented with the following screen:

![2020-01-13_08-26-55](https://user-images.githubusercontent.com/8611594/72273025-762e9e80-35de-11ea-9053-b888e3c927c6.png)

- What version of Tableau are you using?

**Comment by andela-Kshittu** [comment]
@Etroid Would be nice to know what the cause is, My team is experiencing the same issue with our data-connector. It's reproducible with the latest version of tableau public.

**Comment by Etroid** [comment]
@andela-Kshittu are you have the same issue with another data-connector? If so I would suggest opening an issue here: https://github.com/tableau/webdataconnector/issues

**Comment by andela-Kshittu** [comment]
@Etroid Yes I'm having the same issue with our data-connector here https://github.com/datadotworld/tableau-connector, I will go ahead and open an issue ass suggested. Thanks.
